### ISSUE TITLE
Problem when editing the same file in two tabs

### ISSUE THREAD

**Issue by Unarelith** [LeBarbu/vim-epitech#11]
When I edit the same C file in two different tabs of the same vim instance, one of the tabs neither have the red line at 81th column, nor the visual representation for tabs

**Comment by casimir** [comment]
Which version of Vim are you using?

**Comment by Unarelith** [comment]
I'm using version 7.4.944

**Comment by casimir** [comment]
Thinking about it this plugin doesn't play with UI. Most likely this is your distro configuration or your theme doing that. Focused vs unfocused colours.

**Comment by Unarelith** [comment]
I think this is not the problem here, I recently noticed that it doesn't always happens, it's quite random, or I don't see the exact steps to reproduce the bug :/

**Comment by Le-Bit** [comment]
I think i found a way to reproduce: is it when you create a file using vim ?

**Comment by Unarelith** [comment]
No, but I found something. When I launched vim and did ':e truc.c', ':tabnew', ':e truc.c', the second tab had the problem.

**Comment by Le-Bit** [comment]
Well, the steps you described works as intended for me, using either vim 7.4 or neovim 0.1.4

**Comment by Unarelith** [comment]
I recorded myself doing these steps: https://youtu.be/Hzm8frqzOYA

**Comment by deadbaed** [comment]
have you tried again @Quent42340 ?
